cdc/redo/writer/file.go

```diff
@@ -403,56 +399,84 @@ func (w *Writer) rotate() error {
 	return w.openNew()
 }
 
-// GC implement GC interface
+// GC implement GC interface, if checkPointTs == math.MaxInt32 means remove all logs
 func (w *Writer) GC(checkPointTs uint64) error {
-	if !w.IsRunning() || w.isGCRunning() {
+	if w.isGCRunning() {
 		return nil
 	}
 
 	w.gcRunning.Store(true)
 	defer w.gcRunning.Store(false)
 
+	// if checkPointTs == math.MaxInt32 means remove all logs, stop write first
+	if checkPointTs == math.MaxInt32 {
```

Comment:
Better to add a constant variable alias.

Reply:
done